tests/parametric/test_headers_precedence.py

```diff
@@ -640,7 +640,7 @@ def test_headers_precedence_propagationstyle_datadog_tracecontext(self, test_age
         assert "tracestate" in headers6
         assert len(tracestate6Arr) == 1 and tracestate6Arr[0].startswith("dd=")
 
-    @missing_feature(context.library == "java", reason="not_implemented yet")
+    @irrelevant(context.library < "java@v1.43.0", reason="not implemented yet")
```

Comment:
I think the good marker here is `missing_feature`, why do you think it should be `irrelevant` ? 🤔